packages/core/resolve-query/src/index.ts

```diff
@@ -81,76 +91,90 @@ const createQuery = ({
     return [options[optionsMap[flag][0]], options[optionsMap[flag][1]]]
   }
 
-  const read = async ({ modelName, jwtToken, ...options }) => {
+  const read = ({
+    modelName,
+    jwt,
+    ...options
+  }: {
+    modelName: string
+    jwt: string
+    options: any[]
+  }): any => {
     checkModelExists(modelName)
     const [modelOptions, modelArgs] = parseOptions(options)
 
-    const result = await models[modelName].read(
-      modelOptions,
-      modelArgs,
-      jwtToken
-    )
-
-    return result
+    return models[modelName].read(modelOptions, modelArgs, jwt)
   }
 
-  const readAndSerialize = async ({ modelName, jwtToken, ...options }) => {
+  const readAndSerialize = ({
```

Comment:
Regressions: missing `await` statement and elimination of `async` keyword causes Stack Trace to be malformed, so `readAndSerialize` will be excluded from stack, which causes hard debugging
See example here https://repl.it/@IhostVlad/InvolvedUnevenTriangles#index.js